controllers/groupsController.js

```diff
@@ -1,10 +1,21 @@
-import { getSession, getChatList, isExists, sendMessage, formatGroup } from './../whatsapp.js'
+import { getSession, getChatList, isExists, sendMessage, formatGroup, getGroupMeta } from './../whatsapp.js'
 import response from './../response.js'
 
 const getList = (req, res) => {
     return response(res, 200, true, '', getChatList(res.locals.sessionId, true))
 }
 
+const getGroupMetaData = async (req, res) => {
+    const session = getSession(res.locals.sessionId)
+    const jid = req.params.jid
+    let data
```

Comment:
You can just use `const` here and all others `data` variable since you didn't change it value anyway.